src/app/utilities/api-clients/user.js

```diff
@@ -12,10 +12,14 @@ export default class user {
         return http.get(`/zebedee/users?email=${email}`);
     }
 
-    static getAll() {
+    static getAllZebedee() {
```

Comment:
Not a change request, but think it may be safer to expand the getAll which was there and put a conditional in so we don't need to worry about making sure we renamed all calls, there is a slight risk some file has imported the 'user' file with a different alias which may mean a 'find all' may miss a few.

e.g.
```JavaScript
static getAll() {
        const config = window.getEnv();
        if (config.enableNewSignIn) {
            return http.get(`/users`, true);
        }
        return http.get(`/zebedee/users`);
    }
```

Also by referencing the feature flag it should make it easier at the end when we clean up and delete the getAllZebedee we do actually delete it as we are likely to do a code base search for the feature flag.

Reply:
I tried this initially and it caused failures in other places. Looks like it was the positioning of the config initialisation that was the problem. Updated now